packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -87,7 +87,9 @@ The authentication transaction [state machine](#transaction-state) can be modifi
 
```

Comment:
This PR is to revert the change under https://github.com/okta/okta-developer-docs/pull/4177.
Per discussion here: https://okta.slack.com/archives/CP4D95VNY/p1686158762481039?thread_ts=1677695447.494849&cid=CP4D95VNY